src/dropbox.ml

```diff
@@ -190,6 +204,16 @@ module type S = sig
                  ?include_deleted: bool -> ?rev: string -> ?locale: string ->
                  ?include_media_info: bool -> ?include_membership: bool ->
                  string -> metadata option Lwt.t
+
+  val delta : ?cursor: string -> ?locale: string -> ?path_prefix: string
+              -> ?include_media_info: bool -> t -> delta Lwt.t
+
+  val latest_cursor : ?path_prefix: string -> ?include_media_info: bool
+                      -> t -> delta Lwt.t
```

Comment:
Indeed, I thought it would be a gain of place to take the same dictionary as delta function.